FRC1710-2019/src/main/java/frc/robot/intakePOS.java

```diff
@@ -0,0 +1,91 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+package frc.robot;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+import edu.wpi.first.wpilibj.Joystick;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class intakePOS extends Command {
+
+  public static TalonSRX IntakeArmAxsisR;
+  public static TalonSRX IntakeArmAxsisL;
+  
+  private static final int LEFT_ID = 5; //Replace with current id
+  private static final int RIGHT_ID = 6; //Replace with current id
+
+  private double AxisPOS = 0.0; // the axis position of just one of the motors
```

Comment:
I'm not sure using POS in the variable name is the best. Makes me think it might be some acronym: https://www.urbandictionary.com/define.php?term=POS

Can we rename this to _axisPosition_?